nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -16,6 +16,9 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [ucarp](https://download.pureftpd.org/pub/ucarp/README), an userspace implementation of the Common Address Redundancy Protocol (CARP). Available as [networking.ucarp](options.html#opt-networking.ucarp.enable).
 
+- [vikunja](https://vikunja.io), an to-do list app. Available as
+  [services.vikunja](options.html#opt-services.vikunja.enable).
```

Comment:
Are you sure? It works for me when I build the docs.

Reply:
> It works but it won't check for typos if you use anything other than an
anchor.

Then I don't understand why the other people used it in this file.

Reply:
I think that you're wrong. This are the release notes, not the (module) options. So I need to link to "options.html#...".

Reply:
The documentation system supports two kinds of links – internal and external. When you use just the hash fragment, it is considered an internal link that will be resolved to an anchor anywhere in the book. If you use file name, it will be considered an external link.

Reply:
Okay, thank you and sorry for not changing it before. Now I changed it in fc5dc6d77f4bee73a9b4fcc1a10c0871e240fcff.